desktop/core/src/desktop/lib/rest/raz_http_client.py

```diff
@@ -45,12 +45,11 @@ def execute(self, http_method, path, params=None, data=None, headers=None, allow
 
     response = raz_client.get_url(action=http_method, path=url, headers=headers)
 
-    signed_path = path + ('?' if '?' not in url else '&') + response['token']
+    signed_path = url + ('?' if '?' not in url else '&') + response['token']
 
     return super(RazHttpClient, self).execute(
         http_method=http_method,
         path=signed_path,
-        params=params,
```

Comment:
Ha! Indeed looks like we already use them in _make_url()

Reply:
Agreed!